shared/lib/deep-links/routes/index.ts

```diff
@@ -34,3 +35,4 @@ addRoute(home);
 addRoute(notifications);
 addRoute(swap);
 addRoute(nonevm);
+addRoute(perps);
```

Comment:
If you want to write a test ensuring this isn't ever accidentally broken you can copy the test `"handles /buy route redirect"` from `test/e2e/tests/deep-link/deep-link.spec.ts` and change `buy` to `perps` and the destination URL used in the test to the expected value.